### DISCUSSION TITLE
Strapi version for strapi-plugin-rest-cache

### DISCUSSION THREAD

**Discussion by marianamohr** [strapi-community/strapi-plugin-rest-cache#73] [Q&A] [answered]
Hi Folks!

I'm trying to install strapi-plugin-rest-cache version 4.2.8 on a strapi I'm using with version 4.8.2.

My steps were:

yarn add strapi-plugin-rest-cache

Add plugins.ts to config/env/development.

but it did not work. I thought, if the strapi version was not compatible with strapi-plugin-rest-cache.

This can happen?

**Comment by derrickmehaffy** [comment, marked as answer]
You shouldn't use the env folder with the development environment for development it should just be ./config/plugins.ts

Fairly certain this is a Strapi bug.

**Reply by marianamohr** [reply]
thanks @derrickmehaffy !!

**Reply by derrickmehaffy** [reply]
Quite welcome 🤗
